python/ray/rllib/tests/test_lstm.py

```diff
@@ -138,11 +138,12 @@ def __init__(self):
 
     def reset(self):
         self.i = 0
-        return [self.i]
+        return np.array([self.i], dtype=np.float32)
```

Comment:
I think we want to support returning a plain python array -- could that be allowed?

Reply:
whether plain list is allowed depends on the which gym space type the `observation_space` is. For some space types, there `contains()` method just check the range of each entry. For others, the `contains()` check `shape` at first. As you know, only numpy array has `shape`. So, what's your opinion?

Reply:
Maybe it's ok, we can try to change this and see if anyone complains. It is better to return an array with known dtype anyways.